src/mscorlib/shared/System/Text/StringBuilder.cs

```diff
@@ -1645,6 +1645,34 @@ public bool Equals(StringBuilder sb)
             }
         }
 
+        /// <summary>
+        /// Determines if the contents of this builder are equal to the contents of ReadOnlySpan<char>.
+        /// </summary>
+        /// <param name="value">The ReadOnlySpan<char>.</param>
+        public bool Equals(ReadOnlySpan<char> value)
+        {
+            if (value.Length != Length)
+                return false;
+
+            StringBuilder thisChunk = this;
+            int offset = 0;
+            int chunk_length = -1;
```

Comment:
This doesn't need to be initialized to -1 (the value won't ever be used).  It can also just be declared in the loop rather than here.